user-storage-simple/README.md

```diff
@@ -60,13 +67,6 @@ admin console or on the registration pages of <span>Keycloak</span>, will be cre
 to the Users tab in the Admin Console and create a new user, you'll be able to see the provider in action.  You can also
 edit the file yourself to add the username/password pairs you want.
 
-Integration test of the Quickstart
-----------------------------------
-
-1. Make sure you have an Keycloak server running with an admin user in the `master` realm or use the provided docker image
-2. You need to have Chrome browser installed and updated to the latest version.
-3. Run `mvn test -Pkeycloak-remote`
-
```

Comment:
@pedro-hos I've checked it and I can confirm these steps are still valid. Could you revert this change? 

I'd just add to the first point that keycloak has to be listening on `http://localhost:8180/auth`.

Then `createTestArchive` method could be removed from `ArquillianSimpleStorageTest`.

Reply:
And on top of that if you could cherry-pick this [commit](https://github.com/keycloak/keycloak-quickstarts/commit/f9fd95a03c3a683399d8624a49a69c60e77843a3), it'd add the intergation test in GHA. Thanks

Reply:
@vramik I have finished the changes! Thanks